server/service/getcars.ts

```diff
@@ -0,0 +1,47 @@
+import {db} from './db';
+import RowDataPacket from "mysql2/typings/mysql/lib/protocol/packets/RowDataPacket";
+import OkPacket from "mysql2/typings/mysql/lib/protocol/packets/OkPacket";
+import ResultSetHeader from "mysql2/typings/mysql/lib/protocol/packets/ResultSetHeader";
+import {Express} from "express";
+
+
+
+interface Cars {
+    brands?: (RowDataPacket[])[] | RowDataPacket[] | OkPacket | OkPacket[] | ResultSetHeader
```

Comment:
brands: string[] is should be like that. When I read the code I should know what type of data your fields

Reply:
Create separate file for interfaces